apps/web/src/app/[lang]/app/[type]/crm/[data]/(alternative)/[id]/form.tsx

```diff
@@ -168,15 +159,18 @@ export default function Form({
     }
   }
 
-  async function getIndividualsOfMerchant() {
+  async function getIndividualsOfMerchant(page: number) {
     setLoading(true);
     try {
-      const response = await getIndividualByMerchantId();
+      const response = await getIndividualByMerchantId({
+        maxResultCount: 10,
+        skipCount: page * 10,
+      });
       if (response.type === "error") {
         toast.error(response.status);
         return;
       }
```

Comment:
We can handle all error cases here with a code similar to: 
```suggestion
   if (response.type === "error" || response.type === "api-error") {
        toast.error(response.status);
        return;
      }
```

then ues the data returned from the function like the next line, without the need to use the keyword `as`
```
   
      const Individualsdata = response.data;
```
it would be better to handle the api error based on it's type.